### ISSUE TITLE
Continuous Recordings 

### ISSUE THREAD

**Issue by aj-bayanat** [roflcoopter/viseron#465]
I am not able to figure out the configuration to record continuously. Usually with this option you have to provide ffmpeg a length to record each segment like 15mins or so. Could you help me configure this please ?

**Comment by roflcoopter** [comment]
24/7 recordings are not supported yet, but i plan to implement it soon.
If all you want is to record everything at all times you can probably get creative with the configuration, but it would be hard to find the actual events (motion, objects) etc since there is no way to show that as of now

**Comment by aj-bayanat** [comment]
I see, I would rather wait for you to implement it than to figure out a half baked way to do it myself. Thanks for all the work !

**Comment by ktetreault** [comment]
I would be interested as well, have you thought about implementation as far as how it would go?

**Comment by roflcoopter** [comment]
Yes I am working hard on it! You can follow the progress here: #619 

Its a lot of work tho so it will still take me some time before it is ready to test.

I posted some screenshots of the current state here: https://github.com/roflcoopter/viseron/discussions/668#discussioncomment-8029723

**Comment by roflcoopter** [comment]
Closing as this is available since v3 and up
